frontend/libretro.c

```diff
@@ -1652,6 +1658,8 @@ void retro_deinit(void)
 	free(vout_buf);
 #endif
 	vout_buf = NULL;
+  
+  deinit_vita_mmap();
```

Comment:
need not `#ifdef VITA` ? i think, non vita platform will crash or get compile error.